pkg/util/validation/stack.go

```diff
@@ -28,7 +28,17 @@ func validateStackName(s string) error {
 	if stackNameRE.MatchString(s) {
 		return nil
 	}
-	return errors.New("a stack name may only contain alphanumeric, hyphens, underscores, or periods")
+	return errors.New("stack names may only contain alphanumeric, hyphens, underscores, or periods")
```

Comment:
FYI: I'm not sure we want to make this change. As even if the wording reads better, it's possible that <random test in random repo, etc.> may fail by making an innocuous string change like this.

Given that that's probably not the case, if you have a specific example where rewording the error does make things clearly better then we should make the change. But without a specific motivation for making the change, let's not change it for change's sake.